pkg/postgres/migrations/202501061613_reindexTestnetForStaterootChange/up.go

```diff
@@ -0,0 +1,40 @@
+package _202501061613_reindexTestnetForStaterootChange
+
+import (
+	"database/sql"
+	"github.com/Layr-Labs/sidecar/internal/config"
+	"gorm.io/gorm"
+)
+
+type Migration struct {
+}
+
+// Up
+// In response to the rewards-v2 audit, we needed to change how a couple of model state roots
+// were calculated. While this doesnt affect rewards generation, it does affect state roots that
+// have already been calculated and stored for preprod and testnet. To ensure that all state roots
+// are consistent, we need to prune back to the block where rewards-v2 was first deployed to allow
+// rewards to be recaclulated with the new state root calculation.
+func (m *Migration) Up(db *sql.DB, grm *gorm.DB, cfg *config.Config) error {
+	var blockNumber uint64
+	if cfg.Chain == config.Chain_Preprod {
+		blockNumber = 2919262
+	} else if cfg.Chain == config.Chain_Holesky {
+		blockNumber = 2930139
```

Comment:
This block number is correct. 

Reference:
1. https://github.com/Layr-Labs/eigenlayer-contracts-metadata/blob/master/deploys/testnet/2024-12-13-14-11-v0.5.3-rewardsv2/2/multisig.transaction.json
2. https://holesky.etherscan.io/tx/0xeb12a6ffbcbd24924566815ab4b1ac21a99717fea0ba4355db0b443d95a8673b